lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -183,6 +183,9 @@ public Map<String, Object> handleRequest(JourneyRequest input, Context context)
 
             String govukSigninJourneyId = clientOAuthSessionItem.getGovukSigninJourneyId();
 
+            List<Vot> vot = clientOAuthSessionItem.getRequestedVotsByStrength();
+            Vot minimumRequestedVotsByStrength = vot.isEmpty() ? Vot.P2 : vot.get(0);
```

Comment:
Mike has added a method `getLowestStrengthRequestedGpg45Vot` in #2130 which we probably want to use here as well? might want to copy it over or hold until that PR is in cc @DanCorderIPV